options.go

```diff
@@ -125,7 +123,9 @@ type Options struct {
 	// execute it after successful startup.
 	PostStartScriptPath string
 	// Logger is the logger to use for all operations.
-	Logger LoggerFunc
+	Logger eblog.LogFunc
```

Comment:
nit: my brain read it as e-blog. I'm thinking whether we should stick to normal package names like `util`, `log`.

Reply:
This was just my brain on auto-pilot, will rename :+1: